app/models/travis_pro_project.rb

```diff
@@ -10,7 +10,7 @@ def self.project_specific_attributes
   # Add ?token= or &token= to the feed_url, as appropriate
   def feed_url
     URI.parse(super).tap do |uri|
-      params = URI.decode_www_form(uri.query || []) << ['token', ci_auth_token]
+      params = URI.decode_www_form(uri.query || '') << ['token', ci_auth_token]
```

Comment:
[travis_pro_project_spec.rb#L9] (https://github.com/pivotal/projectmonitor/blob/master/spec/models/travis_pro_project_spec.rb#L9) covers this: ie. it failed after upgrading the Gems.